static/css/tools/stat_var_widget.scss

```diff
@@ -97,3 +97,7 @@ $transparent-red: rgba(240, 230, 231, 0.2);
   cursor: pointer;
   background-color: $transparent-gray;
 }
+
+#hierarchy-section .node-no-data {
```

Comment:
looks like .node-no-data with opacity 30% is also set in stat_var_hierarchy.tsx. Does it need to be set in both places?

Reply:
Removed it here.